### DISCUSSION TITLE
Samsung U32J58X via HDMI port

### DISCUSSION THREAD

**Discussion by JJebara** [MonitorControl/MonitorControl#1167] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Volume does not work, it shows but doesn't work. 

### Expected behavior

to be able to control the volume of speaker connected to the monitor. 
Macbook pro connected via HDMI to the screen. 

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.5
- Mac model: Macbook Pro 14-inc 2021 Apple M1 Pro
- MonitorControl version: 4.10.0 Build 7034
- Monitor(s): U32J58X
- Monitor Cable(s)/Connection(s): hdmi 
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
The HDMI port of Macs released after 2018 do not support DDC. I recommend using USB-C instead. #750

**Reply by JJebara** [reply]
I tried using USB-C on LG 4K and that didn't work either. What do you suggest I should do to troubleshoot this issue.

**Reply by waydabber** [reply]
You can try to see if brightness control via DDC works (brightness changes in the 50-100% range). If it does but volume doesn't then DDC should work all right, but the display firmware does not support volume control via DDC - many displays are like this. Most LGs should work though.
